app/views/users/edit.mobile.haml

```diff
@@ -0,0 +1,2 @@
+.settings_container
+  = render "edit", mobile: true
```

Comment:
Hmm ? When has this change ? I used it to render a small block on mobile view. As I recall, it was the the password fied above the close account button that was present in mobile view and not on desktop. That section did not have the same class names and IDs than desktop and I kept it as is to avoid having to modify the JS.